recipe/meta.yaml

```
@@ -155,7 +155,12 @@ outputs:
         - "/opt/intel/compilers_and_libraries/linux/mpi/intel64/lib"
     requirements:
       run:
-        - intel-openmp {{ version.split('.')[0] }}.*
+        # Due to issues with Intel's repack of their oneAPI binaries for OSX,
+        # we could not update intel-openmp to the 2023 version. Therefore, for
+        # OSX we use a slightly older version. This works perfectly fine but
+        # we should get rid of it whenever we have a newer version.
+        - intel-openmp {{ version.split('.')[0] }}.*              # [not osx]
+        - intel-openmp                                            # [osx]
```

Since it's a run dependency, could we be more restrictive and set 
```suggestion
        - intel-openmp 2022.*,2023.*                              # [osx]
```